### ISSUE TITLE
Naming EFS volume with docker-compose

### ISSUE THREAD

**Issue by jbye** [ContainX/docker-volume-netshare#62]
I browsed through issue #21 and saw the use of the "share" driver option in use for nfs, to get the plugin to work nicely with docker-compose v2. 

There does not seem to be an equivalent for EFS? How can I create a volume, using docker-compose, with the EFS driver? The plugin keeps trying to resolve dns using the docker-compose volume name, e.g.: Mounting EFS volume eu-west-1a.some_compose_service_fs-58xxxxxxx.efs.eu-west-1.amazonaws.com: on /var/lib/docker-volumes/netshare/efs/some_compose_service_fs-58xxxxxxx

**Comment by gondor** [comment]
Fixed in Release 0.32

**Comment by florentb** [comment]
Just wanted to share my experience on this.

I was using `v0.18` with `Docker 1.12.1` and my `Docker Compose` setup was: 

```yaml
version: '2'

volumes:
  data:
    driver: efs
    driver_opts:
      share: "fs-xxxxxxxx"

services:
  service:
    image: test
    volumes:
    - data:/test   
```

This setup was convenient cause it lets me name my volumes.

Today, while I was publishing a new app, I realized I couldn't do this anymore with an up to date stack (`v0.32` with `Docker 1.12.3`) because as far as I understand, Docker Volume Netshare try to extract the EFS ID from the volume name: 

```
time="2016-11-14T15:35:30Z" level=info msg="Starting EFS :: availability-zone: , resolve: false, ns: 169.254.169.253" 
time="2016-11-14T15:36:13Z" level=error msg="Error during resolve: Couldn't resolve name 'eu-west-1a..efs.eu-west-1.amazonaws.com.' : dns: bad rdata"
time="2016-11-14T15:36:13Z" level=info msg="Mounting EFS volume eu-west-1a..efs.eu-west-1.amazonaws.com: on /var/lib/docker-volumes/netshare/efs/test_data" 
2016/11/14 15:36:13 mount.nfs4: Failed to resolve server eu-west-1a..efs.eu-west-1.amazonaws.com: Name or service not known
```

So I changed my as follow:

```yaml
version: '2'

volumes:
  fs-xxxxxxxx:
    driver: efs

services:
  service:
    image: test
    volumes:
    - fs-xxxxxxxx:/test   
```

...and it seems to work.

@gondor @hyperius Is this the "official" way to do this? Do I missed something?
